app/src/test/java/org/mozilla/focus/settings/SearchEngineValidationTest.kt

```diff
@@ -0,0 +1,71 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.focus.settings
+
+import okhttp3.mockwebserver.MockResponse
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.robolectric.RobolectricTestRunner
+
+import okhttp3.mockwebserver.MockWebServer
+import org.junit.Assert.assertFalse
+import org.junit.Assert.assertTrue
+import org.mozilla.focus.settings.ManualAddSearchEngineSettingsFragment.isValidSearchQueryURL
+
+@RunWith(RobolectricTestRunner::class)
+// This unit test is not running on an Android device. Allow me to use spaces in function names.
+@Suppress("IllegalIdentifier")
+class SearchEngineValidationTest {
+    @Test
+    fun `URL returning 200 OK is valid`() = withMockWebServer(responseWithStatus(200)) {
```

Comment:
Interesting! It's pretty sweet we get one of the biggest benefits of behavioral unit tests (readability) without the libraries/boilerplate code.